txnkv/client.go

```diff
@@ -93,6 +99,8 @@ func NewClient(pdAddrs []string, opts ...ClientOpt) (*Client, error) {
 		if err != nil {
 			return nil, err
 		}
+		// Set etcd namespace prefix if we are using api v2.
+		namespacePrefix = fmt.Sprintf(tidbKeyspaceEtcdPathPrefix+"%d", codecCli.GetCodec().GetKeyspaceID())
```

Comment:
How about making the namespace prefix an option for `NewClient`, this hard-coded path might couple the logic with TiDB.

Reply:
done, PTAL